Resources/Prototypes/_StarLight/Catalog/Fills/Crates/cargo.yml

```
@@ -20,4 +20,97 @@
     - id: ClothingOuterHardsuitSpaceHazard
     - id: ClothingMaskBreath
     - id: OxygenTankFilled
-    - id: NitrogenTankFilled
\ No newline at end of file
+    - id: NitrogenTankFilled
+
+- type: entity
+  id: CrateBrandedPlushies
+  parent: CrateGenericSteel
+  name: Crew Plushie Crate
```

```suggestion
  name: crew plushie crate
```

Dont capitalize please